vault/changelog.release.2022-08-30.md

```diff
@@ -0,0 +1,74 @@
+---
+id: 2fbbket7taz53ocm9o1s7ht
+title: 0.110
+desc: ''
+updated: 1661810898365
+created: 1661784579096
+---
+
+Dendron 0.110 has sprouted  🌱
+August 30, 2022
+
+
+## Highlights
+
+- feat(workspace): introduce sqlite as a plugabble metadata store (#3401) @kevin [docs](https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.topic.workspace.sqlite.md#L9:L9)
```

Comment:
can you replace with the following?

**SQLite for Metadata Support**: Dendron now supports using SQLite as an backend for metadata. 

As a refresher, Dendron currently creates a [[cache|dendron://dendron.dendron-site/dendron._ref.caching]] of your workspace metadata as a local json file. With this change, you can opt-in to use sqlite as the backend cache. 

![[dendron://dendron.dendron-site/dendron.topic.workspace.sqlite#details]]